src/utils.js

```diff
@@ -658,7 +658,9 @@ export default {
         } else if (def.type === 'hasOne') {
           if (localKey) {
             prop.get = function () {
-              return get(this, localKey) ? definition.getResource(relationName).get(get(this, localKey)) : undefined
+              let key = get(this, localKey);
```

Comment:
The value of `key` here is either going to be a string, a number, `null` or `undefined`.